apps/polyflix/src/modules/studio/pages/videos/videos-list.page.tsx

```diff
@@ -73,15 +78,61 @@ export const VideosListPage = () => {
                 </IconButton>
               }
             >
-              <ListItemButton
-                component={RouterLink}
-                to={polyfilxRouter().studio.videos.view(video?.id)}
+              <Tooltip
+                title={'ID: ' + video.id}
+                placement="top"
+                arrow
+                disableFocusListener
               >
-                <ListItemAvatar>
-                  <Icon name="eva:npm-fill" />
-                </ListItemAvatar>
-                <ListItemText primary={video.title} />
-              </ListItemButton>
+                <ListItemButton
+                  component={RouterLink}
+                  to={polyflixRouter().studio.videos.view(video?.id)}
+                >
+                  <Box
+                    width={'55px'}
+                    height={'30px'}
+                    borderRadius={'3px'}
+                    overflow="hidden"
+                    marginRight={1}
+                  >
+                    <ImageCover ratio={16 / 9} src={video.thumbnail} />
+                  </Box>
+
+                  <Stack direction="column" spacing={0}>
+                    <Typography variant="body1" display="flex" lineHeight={1}>
+                      {video.title}
+                      <Typography sx={{ fontSize: '0.8em', marginLeft: 1 }}>
+                        {t('common.informations.publish')}{' '}
```

Comment:
```suggestion
                        {t('common.informations.publish')}
```

Reply:
We need this space between i18n text and user firstname